app/helpers/errors_helper.rb

```diff
@@ -1,25 +1,16 @@
 module ErrorsHelper
-  def errors_notification resource
-    return "" unless !resource.errors.empty?
+  def help_error_message(resource, attribute, default = "")
+    has_error = resource.errors.details.include? attribute
+    error_msgs = resource.errors.full_messages_for(attribute).join(". ")
 
-    messages = resource.errors.full_messages.map { |msg| content_tag(:li, msg) }.join
-    sentence = I18n.t("errors.messages.not_saved",
-                      :count => resource.errors.count,
-                      :resource => resource.class.model_name.human.downcase)
+    return "" unless has_error || default
 
-    html = <<-HTML
-    <div class="notification is-warning">
-      <p>#{sentence}</p>
-      <ul>#{messages}</ul>
-    </div>
-    HTML
-
-    html.html_safe
+    content_tag :p, has_error ? error_msgs : default, class: "help#{has_error ? " is-danger" : ""}"
   end
 
-  def help_error_message resource, attribute
-    if resource.errors.details[attribute]
-      "<p class=\"help is-danger\">#{resource.errors.full_messages_for(attribute).join('. ')}</p>".html_safe
-    end
+  def errors_count(resource)
+    I18n.t("errors.messages.not_saved",
```

Comment:
Do you need `I18n` here? I know you need it in tests, but as far as I know you don't need it elsewhere.

I wouldn't be surprised if you need it in helpers, as it could be that models, controllers etc have it as a global.